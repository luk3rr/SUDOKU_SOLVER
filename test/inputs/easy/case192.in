000908000 043000108 700000002 020890050 400050007 930007010 000000601 000000070 002001003 
