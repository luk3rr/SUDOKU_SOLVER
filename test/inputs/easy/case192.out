Solving the following grid:
0 0 0 | 9 0 8 | 0 0 0 
0 4 3 | 0 0 0 | 1 0 8 
7 0 0 | 0 0 0 | 0 0 2 
------+-------+------
0 2 0 | 8 9 0 | 0 5 0 
4 0 0 | 0 5 0 | 0 0 7 
9 3 0 | 0 0 7 | 0 1 0 
------+-------+------
0 0 0 | 0 0 0 | 6 0 1 
0 0 0 | 0 0 0 | 0 7 0 
0 0 2 | 0 0 1 | 0 0 3 


Solution found :')

2 1 6 | 9 4 8 | 7 3 5 
5 4 3 | 6 7 2 | 1 9 8 
7 8 9 | 3 1 5 | 4 6 2 
------+-------+------
1 2 7 | 8 9 4 | 3 5 6 
4 6 8 | 1 5 3 | 9 2 7 
9 3 5 | 2 6 7 | 8 1 4 
------+-------+------
3 7 4 | 5 2 9 | 6 8 1 
8 5 1 | 4 3 6 | 2 7 9 
6 9 2 | 7 8 1 | 5 4 3 

Algorithm: BFS
Total time: 71 ms
Total expanded states: 11632
